templates/zerver/no_auth_backend.html

```diff
@@ -0,0 +1,10 @@
+{% extends "zerver/portico.html" %}
+{% block portico_content %}
+
+<h1>No Auth Backend Enabled</h1>
+
+<p>
+    TODO put instructions on how to set up an auth backend here.
```

Comment:
I think we can mostly just refer the sysadmin to http://zulip.readthedocs.io/en/latest/prod-install.html#step-3-configure-zulip